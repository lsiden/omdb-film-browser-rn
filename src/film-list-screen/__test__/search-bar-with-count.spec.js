import React from "react"
import renderer from "react-native-test-utils"

import SeearchBarWithCount from "../search-bar-with-count"

jest.mock("components/search-bar", () => "SearchBar")

const defaultProps = () => {
  return {
    totalResults: 99
  }
}

const createWrapper = (props = {}) =>
  renderer(<SeearchBarWithCount {...{ ...defaultProps(), ...props }} />)

it("renders list of titles", () => {
  const wrapper = createWrapper()
  expect(wrapper.toJSON()).toMatchSnapshot()
})
