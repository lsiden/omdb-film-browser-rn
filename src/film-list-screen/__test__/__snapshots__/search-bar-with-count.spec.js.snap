// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders list of titles 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "black",
      "borderBottomWidth": 1,
      "borderStyle": "solid",
    }
  }
>
  <SearchBar />
  <View
    style={
      Object {
        "paddingBottom": 2,
        "paddingLeft": 20,
        "paddingTop": 2,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 14,
        }
      }
    >
      Found 99 results.
    </Text>
  </View>
</View>
`;
